### PR TITLE
docs: Add migration information (20.11 to 21.03)

### PR DESCRIPTION
**Pull request by bucanero**
Fixes DOC-229

### CODE REVIEW

File: content/deploy/dgraph-administration.md [hunk 1]
```diff
@@ -534,6 +535,30 @@ are affected. Then, you can drop the old types and predicates from DB.
...
+
+1. Use [binary backup]({{< relref "enterprise-features/binary-backups.md">}}) to export data from the old cluster
+2. Ensure it is successful
+3. [Shutdown Dgraph]({{< relref "#shut-down-database" >}}) and wait for all writes to complete
+4. Upgrade `dgraph` binary to `v21.03.0`
+5. [Restore]({{< relref "enterprise-features/binary-backups.md#restore-from-backup">}}) from the backups using the upgraded `dgraph` binary
+6. Start a new Dgraph cluster using the restored data directories
+7. Upgrade the CORS and persisted queries using the following command:
+    ```sh
+    dgraph upgrade  --from v20.11.0 --to v21.03.0 --user groot --password password --alpha-http http://localhost:8080 --acl --deleteOld
```
Comment (NamanJain8) [inline content/deploy/dgraph-administration.md:548]: Please add one more addition to this,
To upgrade an ACL cluster use
```
dgraph upgrade --from v20.11.0 --to v21.03.0 --user groot --password password --alpha http://localhost:9080 --alpha-http http://localhost:8080 --deleteOld
```
To upgrade non-acl cluster use: 
```
dgraph upgrade --from v20.11.0 --to v21.03.0 --alpha http://localhost:9080 --alpha-http http://localhost:8080 --deleteOld
```

File: content/deploy/dgraph-administration.md [hunk 2]
```diff
@@ -534,6 +535,30 @@ are affected. Then, you can drop the old types and predicates from DB.
...
+    This is required because previously CORS information was stored in `dgraph.cors` predicate which has
+    now been moved to be a part of the GraphQL schema. Also, the format of persisted queries has changed.
+    Some of the internal deprecated predicates will be removed by this change.
+
+    You can use `--dry-run` option in `dgraph upgrade` command to see a dry run of what the upgrade
+    command will do.
+
+{{% notice "note" %}}
+The above steps are valid for migration from a cluster in `v20.11` to a single-tenant cluster in `v21.03`, 
+as backup and restore are cluster-wide operations and a single namespace cannot be restored.
```
Comment (NamanJain8) [inline content/deploy/dgraph-administration.md:559]: A single namespace cannot be restored in a **multi-tenant cluster**.

**Review by NamanJain8 — APPROVED** [review]
Got few comments. Please address them before merging.

File: content/deploy/dgraph-administration.md (hunk 2 above)
Comment (bucanero) [reply content/deploy/dgraph-administration.md:559]: fixed

File: content/deploy/dgraph-administration.md (hunk 1 above)
Comment (bucanero) [reply content/deploy/dgraph-administration.md]: fixed

**Pull request merged**